openpype/hosts/max/plugins/create/create_review.py

```diff
@@ -12,6 +12,32 @@ class CreateReview(plugin.MaxCreator):
     family = "review"
     icon = "video-camera"
 
+    review_width = 1920
+    review_height = 1080
+    percentSize = 100
+    keep_images = False
+    image_format = "png"
+    visual_style = "Realistic"
+    viewport_preset = "Quality"
+    vp_texture = True
+    anti_aliasing = None
```

Comment:
Pretty sure this should default to `"None"`

```suggestion
    anti_aliasing = "None"
```